prisma/schema.prisma

```diff
@@ -77,36 +77,38 @@ model PairLiquidityInfoHistoryError {
 }
 
 model PairLiquidityInfoHistoryV2 {
+  id               Int      @id @default(autoincrement())
+  pair             Pair     @relation(fields: [pairId], references: [id])
+  pairId           Int
+  eventType        String
+  reserve0         Decimal  @db.Decimal(100, 0)
+  reserve1         Decimal  @db.Decimal(100, 0)
+  deltaReserve0    Decimal? @db.Decimal(100, 0) // Null if event is a manually triggered Sync event
+  deltaReserve1    Decimal? @db.Decimal(100, 0) // Null if event is a manually triggered Sync event
```

Comment:
we could also make it not optional and use `0` or calculate even calculate it?

Reply:
I'd say `0` makes sense, as during a sole sync event, there is nothing added or removed, what do you think?

Reply:
Sync events are actually triggered to update the balance from the token contract to the pair contract, so if somebody were to send some tokens to the pair, the reserves in the pair would not be updated until a sync is triggered.

Reply:
Good point, so then it would make more sense calculating it IMO.